src/components/error.tsx

```diff
@@ -0,0 +1,16 @@
+import { useContext } from 'react';
+import { MyContext } from './state';
+
+export const Error = () => {
+  const { error } = useContext(MyContext);
+
+  return (
+    <div className="block">
+      <div className="box table-container">
+        <p data-cy="peopleLoadingError" className="has-text-danger">
+          {error || 'Something went wrong'}
```

Comment:
Consider providing a more descriptive default error message. 'Something went wrong' is very generic and might not be helpful for users trying to understand what happened. Tailor the message to the context of the error if possible.